hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -208,6 +208,18 @@
       .sinceVersion("0.11.0")
       .withDocumentation("Table checksum is used to guard against partial writes in HDFS. It is added as the last entry in hoodie.properties and then used to validate while reading table config.");
 
+  public static final ConfigProperty<String> TABLE_METADATA_INDEX_INFLIGHT = ConfigProperty
+      .key("hoodie.table.metadata.index.inflight")
```

Comment:
should we make it plural.
...metadata.indexes.inflight 
or 
... metadata.indices.inflight

Reply:
will do.. `indexes`